react/react1/week3/react1-week3-homework/src/ToDoList/SingleToDo.js

```diff
@@ -0,0 +1,99 @@
+/*
+This component is the child component of ToDoItems.js 
+and it returns a task description , a task due date , 
+THis component has 3 state: the first one (item state) is for editing the task 
+description or date.
+
+The second state is popupButton state with default value of false 
```

Comment:
I think it might be a good exercise to write everything out like this, perhaps I should get my students to do it also :)
That said, developers will most certainly not read this kind of documentation. Try to narrow it down a little, at least